tiledbcontents/tiledbcontents.py

```diff
@@ -386,6 +386,15 @@ def _create_array(self, uri, retry=0):
             namespace = parts[parts_len - 2]
             array_name = parts[parts_len - 1] + "_" + self.id_generator()
 
+            if namespace is not None and \
```

Comment:
Still it seems this message does not propagate to a message box

Reply:
What is propagated? Just a generic error message?

Reply:
Yes and e is None (while until the yield that i can follow is clearly set)